### ISSUE TITLE
Deprecation warnings related to dplyr/tibble packages

### ISSUE THREAD

**Issue by pstraforelli** [mountainMath/cancensus#137]
I'm getting a bunch of deprecation warnings related to usage of `dplyr` and `tibble` packages when fetching information. Here's a reprex (note that I'm hiding my api key). This doesn't impact my work, but worth looking into:

```
library(cancensus)
#> Census data is currently stored temporarily.
#> 
#>  In order to speed up performance, reduce API quota usage, and reduce unnecessary network calls, please set up a persistent cache directory by setting options(cancensus.cache_path = '<path to cancensus cache directory>')
#> 
#>  You may add this option, together with your API key, to your .Rprofile.
my_api_key <- "<HIDDEN>"
options(cancensus.api_key = my_api_key)

all_regions <- list_census_regions("CA16")
#> Querying CensusMapper API for regions data...
all_vars <- list_census_vectors("CA16")

region_age_gender_educ <- get_census(dataset = "CA16", regions = list(C = "01"),
                                     vectors = c("v_CA16_65", "v_CA16_66", 
                                                 "v_CA16_83", "v_CA16_84", "v_CA16_101", "v_CA16_102",
                                                 "v_CA16_119", "v_CA16_120", "v_CA16_137", "v_CA16_138", "v_CA16_155", "v_CA16_156",
                                                 "v_CA16_173", "v_CA16_174", "v_CA16_191", "v_CA16_192", "v_CA16_209", "v_CA16_210",
                                                 "v_CA16_227", "v_CA16_228", "v_CA16_245", "v_CA16_246",
                                                 "v_CA16_5055", "v_CA16_5056",
                                                 "v_CA16_5058", "v_CA16_5059",
                                                 "v_CA16_5064", "v_CA16_5065", 
                                                 "v_CA16_5073", "v_CA16_5074",
                                                 "v_CA16_5076", "v_CA16_5077",
                                                 "v_CA16_5079", "v_CA16_5080"),
                                     level = "PR")
#> Census data is currently stored temporarily.
#> 
#>  In order to speed up performance, reduce API quota usage, and reduce unnecessary network calls, please set up a persistent cache directory by setting options(cancensus.cache_path = '<path to cancensus cache directory>')
#> 
#>  You may add this option, together with your API key, to your .Rprofile.
#> Querying CensusMapper API...
#> Downloading: 2.2 kB     Downloading: 2.2 kB     Downloading: 2.2 kB     Downloading: 2.2 kB
#> Warning: `data_frame()` is deprecated as of tibble 1.1.0.
#> Please use `tibble()` instead.
#> This warning is displayed once every 8 hours.
#> Call `lifecycle::last_warnings()` to see where this warning was generated.
#> Warning: `as_data_frame()` is deprecated as of tibble 2.0.0.
#> Please use `as_tibble()` instead.
#> The signature and semantics have changed, see `?as_tibble`.
#> This warning is displayed once every 8 hours.
#> Call `lifecycle::last_warnings()` to see where this warning was generated.
#> Warning: The `x` argument of `as_tibble.matrix()` must have column names if `.name_repair` is omitted as of tibble 2.0.0.
#> Using compatibility `.name_repair`.
#> This warning is displayed once every 8 hours.
#> Call `lifecycle::last_warnings()` to see where this warning was generated.
```

**Comment by mountainMath** [comment]
Thanks. We will remove the deprecated functions and replace them in the next release.

**Comment by dshkol** [comment]
Fixed in #139
